controllers/podchaos/podfailure/types.go

```diff
@@ -57,13 +72,13 @@ type Reconciler struct {
 	Log logr.Logger
 }
 
-// Instance returns the instance of PodChaos
-func (r *Reconciler) Instance() *v1alpha1.PodChaos {
+// Object implements the reconciler.InnerReconciler.Object
+func (r *Reconciler) Object() reconciler.InnerObject {
```

Comment:
why change to Object again?

Reply:
To directly implement reconciler method (Apply / Recover / Object). The previous way is add one middle layer to transform.